### PR TITLE
hpcai OpenSora v1.1 576P training & parallel inference

### PR DESCRIPTION
**Pull request by SamitHuang**
# What does this PR do?

Fixes # (issue)
- VAE cache improvement: 
        - supporting different aspect ratios
        - add fps, ori_size information
- some bug fix

Adds # (feature)
- Support training with vae cache of different sizes
- 576P training script
- Parallel inference

## Before submitting
- [x] This PR fixes a typo or improves the docs (you can dismiss the other checks if that's the case).
- [x] Did you read the [contributor guideline](https://github.com/mindspore-lab/mindone/blob/master/CONTRIBUTING.md)?
- [x] Did you make sure to update the documentation with your changes? E.g. record bug fixes or new features in `What's New`. Here are the
      [documentation guidelines](https://github.com/mindspore-lab/mindcv/wiki/%E6%96%87%E6%A1%A3%E7%BC%96%E5%86%99%E8%A7%84%E8%8C%83)
- [x] Did you build and run the code without any errors?
- [ ] Did you report the running environment (NPU type/MS version) and performance in the doc? (better record it for data loading, model inference, or training tasks)
- [ ] Did you write any new necessary tests?

## Who can review?

Anyone in the community is free to review the PR once the tests have passed. Feel free to tag
members/contributors who may be interested in your PR.

@xxx

### CODE REVIEW

File: examples/opensora_hpcai/configs/opensora-v1-1/inference/t2v.yaml
Comment (hadipash) [inline examples/opensora_hpcai/configs/opensora-v1-1/inference/t2v.yaml:1]: Already added t2v config in PR #520.

File: examples/opensora_hpcai/opensora/datasets/video_dataset_refactored.py [hunk 1]
```diff
@@ -63,6 +64,20 @@ def __init__(
...
 
+        # check vae latent folder
+        if self._vae_latent_folder is not None:
+            self.num_latent_resolution = 1
+            resolution_indicators = glob.glob(os.path.join(self._vae_latent_folder, "latent_*x*"))
+            if len(resolution_indicators) > 1:
+                self.num_latent_resolution = len(resolution_indicators)
+                self.latent_resolution_prefix = resolution_indicators
+                _logger.info("Multi-resolution latents detected: {}".format(self.num_latent_resolution))
+>>>>>>> support multi-resolution training with vae cached
```
Comment (hadipash) [inline examples/opensora_hpcai/opensora/datasets/video_dataset_refactored.py:75]: Delete it

File: examples/opensora_hpcai/opensora/datasets/video_dataset_refactored.py [hunk 2]
```diff
@@ -100,13 +115,25 @@ def _get_item(self, idx: int) -> Tuple[Any, ...]:
...
+            vae_latent_data = np.load(vae_latent_path)
+
+            # get fps from csv, or cached latents, or from original video in order
+            if "fps" in data:  # cache FPS for further iterations
+                data["fps"] = np.array(data["fps"], dtype=np.float32)
+            elif "fps" in vae_latent_data:
+                data["fps"] = np.array(vae_latent_data["fps"], dtype=np.float32)
+            else:
                 with VideoReader(data["video"]) as reader:
                     data["fps"] = self._data[idx]["fps"] = reader.fps
```
Comment (hadipash) [inline examples/opensora_hpcai/opensora/datasets/video_dataset_refactored.py:134]: `data["fps"]` should be a `float32` array

**Review by hadipash — APPROVED** [review]

File: examples/opensora_hpcai/opensora/datasets/video_dataset.py [hunk 3]
```diff
@@ -4,7 +4,9 @@
 import time
 
 import albumentations
```
Comment (zhtmike) [inline examples/opensora_hpcai/opensora/datasets/video_dataset.py:6]: import albumentations
import albumentations as A

import twice

**Review by zhtmike — APPROVED** [review]

File: examples/opensora_hpcai/run_infer_vae_parallel.sh
Comment (HaoyangLee) [inline examples/opensora_hpcai/run_infer_vae_parallel.sh:1]: There is another `scripts/run/run_infer_vae_parallel.sh`. Is this script redudant?

File: examples/opensora_hpcai/opensora/datasets/video_dataset.py (hunk 3 above)
Comment (SamitHuang) [reply examples/opensora_hpcai/opensora/datasets/video_dataset.py]: fixed

File: examples/opensora_hpcai/opensora/datasets/video_dataset_refactored.py (hunk 2 above)
Comment (SamitHuang) [reply examples/opensora_hpcai/opensora/datasets/video_dataset_refactored.py]: fixed

File: examples/opensora_hpcai/opensora/datasets/video_dataset_refactored.py (hunk 1 above)
Comment (SamitHuang) [reply examples/opensora_hpcai/opensora/datasets/video_dataset_refactored.py]: fixed

File: examples/opensora_hpcai/opensora/datasets/video_dataset_refactored.py [hunk 4]
```diff
@@ -63,6 +64,19 @@ def __init__(
...
+            self.num_latent_resolution = 1
+            resolution_indicators = glob.glob(os.path.join(self._vae_latent_folder, "latent_*x*"))
+            if len(resolution_indicators) > 1:
+                self.num_latent_resolution = len(resolution_indicators)
+                self.latent_resolution_prefix = resolution_indicators
+                _logger.info("Multi-resolution latents detected: {}".format(self.num_latent_resolution))
+
+        # prepare replacement data in case the loading of a sample fails
+        self._prev_ok_sample = self._get_replacement()
+        self._require_update_prev = False
```
Comment (hadipash) [inline examples/opensora_hpcai/opensora/datasets/video_dataset_refactored.py:78]: duplicated code

**Review by CaitinZhao — APPROVED** [review]

**Review by geniuspatrick — APPROVED** [review]

File: examples/opensora_hpcai/scripts/inference_i2v.py
Comment (zhtmike) [inline examples/opensora_hpcai/scripts/inference_i2v.py:1]: was deleted in #535

**Pull request merged**